### ISSUE TITLE
Importing variables and breakpoints before USWDS

### ISSUE THREAD

**Issue by annekainicUSDS** [usds/us-forms-system#102]
If I don't import _b-variables.scss and _b-breakpoints.scss before USWDS, I get a scss compilation error. Look into what's going on here because it would be best to import these files after all of the USWDS files.

**Comment by jcmeloni-usds** [comment]
@annekainicUSDS Still a thing?

**Comment by annekainicUSDS** [comment]
@jcmeloni-usds Yep still a thing! Haven't looked at it though; not a priority as what we have now works, I just know it's not ideal.

**Comment by jcmeloni-usds** [comment]
cool cool.

**Comment by dmethvin-gov** [comment]
@annekainicUSDS is this the error you got? 
```js
{
  "status": 1,
  "file": "/Users/DWMethvin/Documents/Projects/us-forms-system/node_modules/uswds/src/stylesheets/elements/_inputs.scss",
  "line": 10,
  "column": 3,
  "message": "Incompatible units: 'rem' and '%'.",
  "formatted": `Error: Incompatible units: 'rem' and '%'.\n
        on line 10 of node_modules/uswds/src/stylesheets/elements/_inputs.scss\n
        from line 24 of src/scss/styles.scss\n
        >>   ($base-font-size * $input-line-height) +\n\n   --^\n`
}
```
It's due to the `em`-vs-`px` mismatch, it's mentioned in [b-breakpoints.scss](https://github.com/usds/us-forms-system/blob/66a3749261061398995af256ec4b79a83b9fa446/src/scss/base/_b-breakpoints.scss#L9-L15).

It should get fixed with #57 when we remove Foundation so I'll close this ticket. Feel free to reopen if you think it needs more investigation.

**Comment by annekainicUSDS** [comment]
Yes that's right. Sounds good to me!
